plugins/outputs/timestream/timestream.go

```diff
@@ -286,7 +286,7 @@ func (t *Timestream) writeToTimestream(writeRecordsInput *timestreamwrite.WriteR
 			t.logWriteToTimestreamError(notFound, writeRecordsInput.TableName)
 			// log error and return error to telegraf to retry in next flush interval
 			// We need this is to avoid data drop when there are no tables present in the database
-			return fmt.Errorf("failed to write to Timestream database '%s' table '%s', Error: '%s'",
+			return fmt.Errorf("failed to write to Timestream database '%s' table '%s', Error: '%w'",
```

Comment:
```suggestion
			return fmt.Errorf("failed to write to Timestream database %q table %q: %w",
```
Same for the others in this file..